### DISCUSSION TITLE
how to merge the camera video stream with the 3D model

### DISCUSSION THREAD

**Discussion by qianxifu** [AR-js-org/AR.js#489] [Help] [answered]
how to merge the camera video stream with the 3D model.
anyone can give me some help?

**Comment by nickw1** [comment, marked as answer]
Hi @qianxifu this is a general A-Frame question, rather than an AR.js specific question.

What you'd need is to set the texture of the model to be sourced from the camera feed.

I would imagine someone has written a component to do this already, so it might be worthwhile looking around for this.

**Comment by qianxifu** [comment]
@nickw1 thank you for your advices
